contents/julia_101.md

```diff
@@ -2,24 +2,36 @@
 
 We should include a nice warning box here saying:
 
----
-**Note**
+> **_NOTE:_**  The note content.
+> In this chapter we will cover the basic of Julia as a programming language.
+> Please note that this is not *strictly necessary* for you to use Julia as a tool of data manipulation and data visualization.
+> Having a basic understanding of Julia will definitely make you more *effective* and *efficient* in using Julia.
+> However, if you prefer to get started straight away, you can jump to Chapter [-@sec:dataframes].
 
-In this chapter we will cover the basic of Julia as a programming language.
-Please note that this is not *strictly necessary* for you to use Julia as a tool of data manipulation and data visualization.
-Having a basic understanding of Julia will definitely make you more *effective* and *efficient* in using Julia.
-However, if you prefer to get started straight away, you can jump to Chapter [-@sec:dataframes_overview].
-
----
-
-
-
-
-## Language Syntax {#subsec:syntax}
+## Language Syntax {#sec:syntax}
 ### Functions {#sec:fun}
 ### For Loop {#sec:for}
 ### While Loop {#sec:while}
 ### Conditional If-Else-Elseif {#sec:conditionals}
+### User-defined Types
+
+```jl
+sc(
+"""
+struct Person
+    name::AbstractString
+    age::Int64
+end
+""")
+```
+
+```jl
+sco(
+"""
+rik = Person("Rik", 27)
```

Comment:
```suggestion
gelman = Person("Gelman", 27)
```
How about Gelman as social science statistician and a famous computer scientist like Judea Pearl or Donald Knuth for IEEE?

Reply:
Or Noam Chomsky as a famous social scientist.